opentelemetry/chart/templates/smon-filelog.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
   annotations:
   name: opentelemetry-collector-logs
   labels:
-    prometheus: "{{ .Values.openTelemetry.prometheus }}"
+    prometheus: infra-frontend
```

Comment:
I would rather have this specced in the values, this is very static and will involve touching soon.